README.md

```diff
@@ -84,6 +84,7 @@ Some presses rely on a different logic:
 - `SimLayerKVPress` ([source](kvpress/presses/simlayerkv_press.py), [paper](https://arxiv.org/abs/2410.13846)): identify "lazy" layers, and apply the StreamingLLM approach to them 
 - `DuoAttentionPress` ([source](kvpress/presses/duo_attention_press.py), [paper](https://arxiv.org/abs/2410.10819)): split heads into retrieval heads (no compression) and streaming heads (StreamingLLM approach)
 - `FinchPress` ([source](kvpress/presses/finch_press.py), [paper](https://direct.mit.edu/tacl/article/doi/10.1162/tacl_a_00716/125280)): similar to SnapKV with a dynamic window size and key value re-rotation
+- `KVzipPress` ([source](kvpress/presses/kvzip_press.py), [paper](https://arxiv.org/abs/2505.23416)): accurately identifies redundant KV pairs through context reconstruction, achieving near-lossless query-agnostic compression at eviction ratios of up to 70–80%.
```

Comment:
Can you rephrase to `identifies redundant KV pairs through context reconstruction. Achieves near-lossless compression at the cost of multiple forward passes`?